IdentityCore/src/cache/MSIDCacheAccessor.h

```diff
@@ -105,4 +105,9 @@
                   context:(id<MSIDRequestContext>)context
                     error:(NSError **)error;
 
+@optional
```

Comment:
why is this optional?

Reply:
I am not sure if we need to add the logic for querying all PRT's for legacy cache accessor so I made it optional. Lets discuss this tomorrow.